### PR TITLE
Fix for Apple MacOS M1

### PR DESCRIPTION
**Pull request by acierto**
fixes: https://github.com/kiali/kiali/issues/5580

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ HELM_IMAGE_REPO_SERVER ?= quay.io/kiali/kiali
 DORP ?= docker
 
 # When building the helm chart, this is the helm version to use
-HELM_VERSION ?= v3.5.2
+HELM_VERSION ?= v3.10.1
```
Comment (jmazzitelli) [inline Makefile:21]: is this bump of helm required for ARM support? If not, it would not be related to this issue and so should not be included.

**Comment by jmazzitelli** [comment]
Please create a Kiali issue and associate it with this PR (i.e. put `fixes: <link to issue>` in the description of this PR). Please do not submit PRs to the Kiali project without an associated issue so we can track issues better. Thanks.

Putting on a DNM label until we get an issue for this PR.

File: Makefile (hunk 1 above)
Comment (acierto) [reply Makefile:21]: Yeah it is, as ARM support started from 3.6.0. and if to bump it, why not to bump to the latest.
Let me know what do you think

**Comment by acierto** [comment]
> Please create a Kiali issue and associate it with this PR (i.e. put `fixes: <link to issue>` in the description of this PR). Please do not submit PRs to the Kiali project without an associated issue so we can track issues better. Thanks.
> 
> Putting on a DNM label until we get an issue for this PR.

Thank you for the comments. Created an issue and associated it with this PR

**Comment by jmazzitelli** [comment]
Works on my machine. Confirmed it pulls down helm 3.10.1, and the build-helm-charts target still works. Nothing breaks. I don't have an ARM machine to test, but I assume that will work since that's the whole point :)

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**